graph/graph.go

```diff
@@ -218,7 +218,7 @@ func Copy(dst Builder, src Graph) {
 	}
 	for _, u := range nodes {
 		for _, v := range src.From(u) {
-			dst.SetEdge(src.Edge(u, v))
+			dst.SetEdge(dst.NewEdge(u, v))
```

Comment:
I've been curious to know, what would happen if we accidentally did this, but for nodes? That is add the `graph.Node` of one graph `G1` to another `G2`, without creating a new dedicated node for `G2`.

Good find with fixing the edges here @kortschak.

Reply:
>I've been curious to know, what would happen if we accidentally did this, but for nodes? That is add the `graph.Node` of one graph `G1` to another `G2`, without creating a new dedicated node for `G2`.

There is no reason not to do this - in fact the original code is not incorrect when the destinations and source match for whether they are directed or undirected. There are subtleties that are touched on in gonum/graph#87 though.

Not my find, #417 was found by @DryHumour.